### PR TITLE
 Update README.md to add a section with a list of features

### PR DESCRIPTION
**Pull request by JovannMC**
title.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,17 +19,29 @@ If you want to contribute to SaladBind, please read our [contributing guide](CON
...
+- Three methods of setting up SaladBind: scanning for the Rig ID, getting it from your Salad Auth token, and entering your Rig ID manually
+- Discord Rich Presence
+- No extra programs needed
+- Set advanced miner arguments
+- Save your advanced miner arguments
+
 ## Installation
 
-Head to our [GitHub Releases](https://github.com/VukkyLtd/SaladBind/releases/latest) page, download the latest release for your operating system, and just run it if you're on Windows.  **BUT KEEP READING!** (see below for macOS and Linux)
+Head to our [GitHub Releases](https://github.com/VukkyLtd/SaladBind/releases/latest) page, download the latest release for your operating system, and just run it if you're on widows. **BUT KEEP READING!** (see below for macOS and Linux
```
Comment (TheBlueBurger) [inline README.md:44]: Spelling mistake

**Review by TheBlueBurger — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JovannMC) [reply README.md:44]: oops

Comment (TheBlueBurger) [reply README.md]: Why did you remove the `)`?

Comment (JovannMC) [reply README.md]: goddamn it i keep forgetting to fix the mistakes when i copied it

Comment (JovannMC) [reply README.md]: this should be it but i probably missed something

**Review by vtheskeleton — APPROVED** [review]
Beep Boop Alvar approves

**Pull request merged**